core/store/models/secrets.go

```diff
@@ -24,6 +24,8 @@ func (s Secret) String() string { return redacted }
 
 func (s Secret) GoString() string { return redacted }
 
+func (s Secret) XXXTestingOnlyString() string { return (string)(s) }
```

Comment:
I could avoid implementing this helper and repeatedly perform the operation to get the value in l273-283 of `config_general_test.go` since it's not too much of a burden and likely won't be used by other tests in the future.

Reply:
This cast is already possible to do directly (and is actually the intended use - it just happens implicitly in most cases IIRC) so this method shouldn't be necessary either way.